src/AWS.Deploy.CLI/ServerMode/Tasks/DeployRecommendationTask.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using AWS.Deploy.Common;
+using AWS.Deploy.Common.Recipes;
+using AWS.Deploy.Orchestration;
+
+namespace AWS.Deploy.CLI.ServerMode.Tasks
+{
+    public class DeployRecommendationTask
+    {
+        private readonly CloudApplication _cloudApplication;
+        private readonly Orchestrator _orchestrator;
+        private readonly Recommendation _selectedRecommendation;
+
+        public DeployRecommendationTask(Orchestrator orchestrator, CloudApplication cloudApplication, Recommendation selectedRecommendation)
+        {
+            _orchestrator = orchestrator;
+            _cloudApplication = cloudApplication;
+            _selectedRecommendation = selectedRecommendation;
+        }
+
+        public async Task Execute()
+        {
+            await CreateDeploymentBundle();
+            await _orchestrator.DeployRecommendation(_cloudApplication, _selectedRecommendation);
+        }
+
+        /// <summary>
+        /// TODO: This needs to be reconciled with "DeployCommand.CreateDeploymentBundle" method
+        /// </summary>
+        /// <returns></returns>
```

Comment:
Can you remove this TODO comment and instead add a backlog item?

Reply:
Done